tests/allowed_signers/good_allowed_signers

```diff
@@ -0,0 +1,5 @@
+# Comments are ignored
+mitchell@confurious.io ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIDO0VQD9TIdICZLWFWwtf7s8/aENve8twGTEmNV0myh5
```

Comment:
Are comments at the end of lines supported?

Reply:
> Empty lines and lines starting with a ‘#’ are ignored as comments.

Not mentioned according to https://man.openbsd.org/ssh-keygen.1

Reply:
Added support for it anyway since `git` seems to support it